### PR TITLE
testcase/task_manager: Add testcases about the broadcast functionality

### PR DESCRIPTION
**Pull request by tyha1115**
Add testcases for added APIs (task_manager_set_broadcast_handler() and task_manager_broadcast()).

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [ebe40be0e7d7aefee55a9cce41c7f5ff3c3a2a46] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [ebe40be0e7d7aefee55a9cce41c7f5ff3c3a2a46] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [662a07263b5fd801972e295f922350a58d9fe65b] -

**Comment by seinfra** [comment]
Target : [90b4beafa1e74cb468d373d8d3d9681de31a7d15] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [90b4beafa1e74cb468d373d8d3d9681de31a7d15] -

File: apps/examples/testcase/ta_tc/task_manager/utc/utc_task_manager_main.c [hunk 1]
```diff
@@ -26,28 +26,73 @@
...
+static int broad_cnt;
 
 static void test_handler(int sig, tm_msg_t *info)
 {
 	flag = !strncmp(info->si_value.sival_ptr, TM_SAMPLE_MSG, strlen(TM_SAMPLE_MSG));
 }
 
+static void test_broadcast_handler(int sig, tm_msg_t *info)
+{
+	int rec = info->si_value.sival_int;
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/task_manager/utc/utc_task_manager_main.c]: In my opinion, message or recv_msg is better than rec or recv.

**Comment by seinfra** [comment]
Target : [01e72f8c3e6dfcb2cfe34cb15b9465d30914029c] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [01e72f8c3e6dfcb2cfe34cb15b9465d30914029c] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [c2496a916e8285c55673f5fd24bea3af46cdde0b] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [c2496a916e8285c55673f5fd24bea3af46cdde0b] - Code Rule Check (C++) OK.

**Pull request closed without merging**